components/ContactCard/ContactCard.module.scss

```diff
@@ -45,30 +60,33 @@
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
+    font-weight: $font-weight-bold;
+    font-size: $font-size-labels;
+    color: $color-blue-500;
+}
+
+.socials {
+    display: flex;
```

Comment:
As I pointed out in Slack, the social contacts alignment issue comes from uneven on the items. In this case it comes from `:last-of-type` rule for the `.link` class. Note that you need to disable that for all cases when the item is displayed inline (e.g. in `socials` and in tablet view for the phone/email section)